data/json/vehicleparts/combustion.json

```diff
@@ -36,6 +36,15 @@
     "//": "30% energy efficiency",
     "description": "A combustion engine.  Burns gasoline fuel from a tank in the vehicle."
   },
+  {
+    "abstract": "gas_turbine_engine",
+    "copy-from": "engine_combustion",
+    "type": "vehicle_part",
+    "fuel_type": "gasoline",
+    "m2c": 60,
```

Comment:
It might be worthwhile to lower the m2c here to 45 or, further limiting the amount of total engine power than can be used for safe velocity.  BUt it's probably not necessary since the speed problems come from the MBT being underweight.

Reply:
I've dropped the light tank down to a single diesel v12.

Most elegant way I can see to fix the MBT weight issue would be to create a heavy_plating_military that doesn't perform a whole lot better but just weighs about 2x as much. I think I'll leave that out of this PR but stick it on a to-do list for making a second pass at armored vehicles in the future.

Reply:
Good solution.